test/project_test.rb

```diff
@@ -32,12 +22,12 @@ def test_find_raises_if_not_found
 
   def test_schema
     project = Wordsmith::Project.find 'test'
-    expected = {a: 'Number', b: 'Number', c: 'Number'}
+    expected = { a: 'Number', b: 'Number', c: 'Number' }
```

Comment:
Don't think you want these spaces here...if it were a block, yes, but it's a hash. Elsewhere you don't have spaces for hashes I don't think, but a global search would clarify this.